src/vs/base/test/browser/ui/grid/grid.test.ts

```diff
@@ -8,19 +8,16 @@ import { Direction, Grid, getRelativeLocation, Orientation, SerializableGrid, IS
 import { TestView, nodesToArrays } from './util';
 import { deepClone } from 'vs/base/common/objects';
 
-suite('Grid', function () {
-	let container: HTMLElement;
+function setupGrid(view: TestView): Grid<TestView> {
+	const grid = new Grid(view);
+	grid.container.style.position = 'absolute';
+	grid.container.style.width = `${800}px`;
+	grid.container.style.height = `${600}px`;
 
-	setup(function () {
-		container = document.createElement('div');
-		container.style.position = 'absolute';
-		container.style.width = `${800}px`;
-		container.style.height = `${600}px`;
-	});
+	return grid;
+}
```

Comment:
You should leave the old behaviour of having a fixed size container. Then, create a grid and append it to the container. You probably still need `setup` and `teardown`.